packages/leva/stories/controlled-inputs.stories.tsx

```diff
@@ -31,6 +30,51 @@ export const ExternalUpdatesWithSet: Story = () => {
   )
 }
 
+export const ExternalUpdatesWithGetAndSet: Story = () => {
+  const [{ counter }, set, get] = useControls(() => ({ counter: 0 }))
+  const [{ counter: counter2, counterB }, set2, get2] = useControls('folder', () => ({
+    counter: 0,
+    folder2: folder({
+      counterB: 0,
+    }),
+  }))
+
+  const onClick = useCallback(() => {
+    set({ counter: get('counter') + 1 })
+  }, [])
```

Comment:
`useCallback` is not useful here and will not generate any performance gains since it's passed to non-react elements. You can remove it.

Reply:
`useCallback` removed. It was just to show the use case of `get` when you can't access the values directly from the current render.